checkout-ui-custom/src/_js/_v.custom.checkout.ui.js

```diff
@@ -823,7 +823,13 @@ class checkoutCustom {
   }
 
   updateLang(orderForm) {
-    this.lang = _locale[orderForm.storePreferencesData.countryCode]
+    const clientLocale = orderForm.clientPreferencesData.locale
+
+    /* eslint eqeqeq: 0 */
+    this.lang = Object.values(_locale).find((country) => country.locale == clientLocale)
+    /* eslint eqeqeq: 0 */
+      ? Object.values(_locale).find((country) => country.locale == clientLocale)
+      : _locale[orderForm.storePreferencesData.country]
```

Comment:
There is no `orderForm.storePreferencesData.country` always will be undefined do you wanted to say `orderForm.storePreferencesData.countryCode`?